packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -664,4 +678,69 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         stripeElement.mount(`#${containerId}`);
         this._isMounted = true;
     }
+
+    private async _checkOrderStatus(): Promise<string | undefined> {
```

Comment:
I believe we need to cover it with unit tests

Reply:
I added unit test for new logic